storage/Devices/BcacheImpl.cc

```diff
@@ -53,17 +54,29 @@ namespace storage
 
     const char* DeviceTraits<Bcache>::classname = "Bcache";
 
+    const vector<string> EnumTraits<CacheMode>::names({
+	"writethrough", "writeback", "writearound", "none"
+    });
 
     Bcache::Impl::Impl(const string& name)
-	: Partitionable::Impl(name)
+	: Partitionable::Impl(name), cache_mode(CacheMode::WRITETHROUGH),
+          sequential_cutoff(8 * MiB), writeback_percent(10)
```

Comment:
Where do the defaults come from?

Reply:
it is bcache defaults for new device.

Reply:
+= `// these are the bcache defaults for a new device`

Reply:
With more testing with bcache with various size I actually found that 10% is correct, but sequential cutoff is dynamic and depends "probably" on caching device size. As now for smaller disk I have default 4 MiB.